samples/isilon_v260_k8s_122.yaml

```diff
@@ -5,15 +5,15 @@ metadata:
   namespace: test-isilon
 spec:
   driver:
-    # Config version for CSI PowerScale v2.3.0 driver
-    configVersion: v2.3.0
+    # Config version for CSI PowerScale v2.6.0 driver
```

Comment:
Same comment as above

Reply:
We need to have K8s 1.22 version for VISA Support. For Powerscale, Min version will be 1.22 and Max will be 1.26.